core/src/main/java/kafka/metrics/FilteringJmxReporter.java

```diff
@@ -51,7 +50,7 @@ public void updatePredicate(Predicate<MetricName> predicate) {
                 } else {
                     super.onMetricRemoved(name);
                 }
-            }
-            );
+            });
     }
+
```

Comment:
@dongjinleekr The empty lines at the end of the class are still there although you removed the formatting directive. Maybe it is the default behavior. Could you try to add it again and set the value to 0?